src/models/Profile.js

```diff
@@ -0,0 +1,49 @@
+export default (sequelize, DataTypes) => {
+  const Profile = sequelize.define('profiles', {
+    userId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      references: {
+        model: 'users',
+        key: 'id',
+      },
+      onUpdate: 'CASCADE',
+      onDelete: 'SET NULL'
+    },
+    phoneNumber: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    departmentId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      references: {
+        model: 'departments',
+        key: 'id',
+      },
+      onUpdate: 'CASCADE',
+      onDelete: 'SET NULL'
+    },
+    role: {
+      type: DataTypes.ENUM('Super Admin', 'Travel Admin', 'Manager', 'Staff'),
+      defaultValue: 'Staff',
+      allowNull: true
+    },
+    jobDescription: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    imageUrl: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    isVerified: {
+      type: DataTypes.BOOLEAN,
+      allowNull: true
+    },
+  });
+  Profile.associate = (models) => {
+    Profile.belongsTo(models.Department, { as: 'theDepartment', foreignKey: 'departmentId' });
```

Comment:
Nice one @funmi5, I don't think this association is correct.  Are you saying every profile belongs to a department? I think it should have been the other way round.

Reply:
No, it's not the other way round, every user / user profile belongs to a department.

Reply:
But shouldn't there be a reference from the departments table to profile and users then?

Reply:
@uthdev that is what `belongs to` is for child to parent, you can read more on Sequelize associations. @uthdev

Reply:
@uthdev I think that association is correct, it means that a profile belongs to a department.

Reply:
I get you. So don't we need an association for `userId` too?